plugins/PrivacyManager/templates/usersOptOut.twig

```diff
@@ -23,19 +23,12 @@
                 {% endif %}
             </p>
 
-            <div piwik-form ng-controller="DoNotTrackPreferenceController as doNotTrack">
-
-                {# {{ {'module':'PrivacyManager','nonce':nonce,'action':action} | urlRewriteWithParameters }}#DNT #}
-                <div piwik-field uicontrol="radio" name="doNotTrack"
-                     ng-model="doNotTrack.enabled"
-                     options="{{ doNotTrackOptions|json_encode }}"
-                     value="{% if dntSupport %}1{% else %}0{% endif %}"
-                     inline-help="{{ 'PrivacyManager_DoNotTrack_Description'|translate|e('html_attr') }}">
-                </div>
-
-                <div piwik-save-button onconfirm="doNotTrack.save()" saving="doNotTrack.isLoading"></div>
-
-            </div>
+      <div
+        vue-entry="PrivacyManager.DoNotTrackPreference"
+        is-super-user="{{ isSuperUser|default(null)|json_encode|e('html_attr') }}"
```

Comment:
Seems the `isSuperUser` property is actually unused and could be removed.

Reply:
:+1: